llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -1554,3 +1554,21 @@ let Predicates = [HasCheri, IsRV64, IsCapMode] in {
 defm : CheriLdPat<load, CLC_128>;
 defm : CheriStPat<store, CSC_128, GPCR>;
 } // Predicates = [HasCheri, IsRV64, IsCapMode]
+
+
+/// CHERI memory colouring extensions
```

Comment:
Instructions go in the first half of the file to keep MC and CodeGen cleanly separated*

(* pseudos are more complicated, but placed based on whether they're used for assembly or just CodeGen)